src/components/NoteDialogComponents/NoteDialogNote.scss

```
@@ -2,39 +2,41 @@
 @import "src/components/Note/Note.scss";
 
 .note-dialog__note {
+  --note-footer-height: 28px;
+
+  display: grid;
+  grid-template-columns: 11fr 1fr;
+  grid-template-rows: 4fr 1fr;
+
   margin-bottom: $margin--default;
   padding: $padding--default $padding--medium;
   position: relative;
   border-radius: $note__border-radius;
   background-color: $color-white;
-  box-shadow: 0 6px 9px 0 $note__box-shadow-color;
   transition: transform 0.3s ease-in-out, box-shadow 0.3s ease-in-out;
   list-style-type: none;
   z-index: $note-dialog-z-index;
   min-width: calc(240px);
   width: calc(100vw - 36px);
   max-width: 380px;
 }
-[theme="dark"] {
+
+@media #{$tablet} {
   .note-dialog__note {
-    background-color: $color-dark-mode-note;
-    box-shadow: 0 6px 9px 0 $note__box-shadow-color-dark-mode;
+    max-width: 415px;
+    width: calc(100vw - 32px);
```

I guess 32px should be the margin right? So 2 times 16px? I'd prefer to use that then.
```suggestion
    width: calc(100vw - (2 * $margin--default));
```
Default margin is 16px, large margin is 32px. Depending on the use case (if its on both sides or just one) pick the respective margin